docker-compose.yml

```diff
@@ -96,5 +101,6 @@ services:
     image: mongo
     volumes:
       - /var/log/multi-cloud:/var/log/multi-cloud
+      - ./log.conf:/etc/log.conf
     ports:

```

Comment:
why need conf in mongo?

Reply:
Mongo didn't need it.
I have deleted the volume mapping in mongo